etc/di.xml

```diff
@@ -30,6 +32,11 @@
     <type name="Hyva\Admin\Block\Adminhtml\HyvaGrid">
         <arguments>
             <argument name="gridTemplate" xsi:type="string">Hyva_Admin::grid.phtml</argument>
+            <argument name="children" xsi:type="array">
+                <item name="navigation" xsi:type="object">Hyva\Admin\Block\Grid\Navigation</item>
+                <item name="actions" xsi:type="object">Hyva\Admin\Block\Grid\Actions</item>
+                <item name="exports" xsi:type="object">Hyva\Admin\Block\Grid\Exports</item>
+            </argument>
```

Comment:
I would really this to be with regular `Template` blocks, in which case the wouldn't have to be injected at all tough, since the could be created on the fly by the grid view model when needed.
I already commended at the top of this PR that I think it would be better to add logic to the view model to retrieve the child html content, rather then add new blocks to the parent block.